src/api/registrationApi.ts

```diff
@@ -71,9 +81,11 @@ export const createDraftDoi = async (registrationId: string) =>
     method: 'POST',
   });
 
-export const fetchRegistration = async (registrationIdentifier: string) => {
+export const fetchRegistration = async (registrationIdentifier: string, shouldNotRedirect?: boolean) => {
   const fetchRegistrationResponse = await apiRequest2<Registration>({
-    url: `${PublicationsApiPath.Registration}/${registrationIdentifier}`,
+    url: shouldNotRedirect
+      ? `${PublicationsApiPath.Registration}/${registrationIdentifier}?doNotRedirect=true`
```

Comment:
får litt spasmer av dette altså, men skal jobbe med å undertrykke dem 🙃

Reply:
Jeg reverserer det, så ser vi om det dukker opp klager på de.

Reply:
Ja, vi backendere får også hodepine når vi leser den tilsvarende "shouldNotRedirect" koden i publication-api.
Antar vi ikke får designpris for denne.